hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -124,6 +124,23 @@ execute_command_in_container(){
   set +e
 }
 
+## @description Stop a list of named containers
+## @param       List of container names, eg datanode_1 datanode_2
+stop_containers() {
+  set -e
+  docker-compose -f "$COMPOSE_FILE" stop $@
+  set +e
+}
+
+
+## @description Start a list of named containers
+## @param       List of container names, eg datanode_1 datanode_2
+start_containers() {
+  set -e
+  docker-compose -f "$COMPOSE_FILE" start $@
```

Comment:
```suggestion
  docker-compose -f "$COMPOSE_FILE" --no-ansi start $@
```

Reply:
Of course, I had noticed this in other commands but forgot to add the switch to my new ones. I will add it in.